tooling/artifact_cli/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "noir_artifact_cli"
```

Comment:
```suggestion
name = "noir-execute"
```

Reply:
I left it like this because as a library it is imported in the next PR by every other CLI that needs to read or write artifacts

Reply:
Ok, we can deal with this later.